sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusConnectionStringProperties.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.amqp.implementation.ConnectionStringProperties;
+
+import java.net.URI;
+
+/**
+ * A utility class that parses a connection string into sections.
```

Comment:
Can you add more documentation on how to use this and what is the expected format?

Reply:
Added more docs

Reply:
Looks great, thanks for adding a very detailed javadoc!